plugins/modules/cloud/alicloud/ali_instance_info.py

```diff
@@ -216,11 +224,6 @@
             returned: always
             type: str
             sample: m-0011223344
-        inner_ip_address:
```

Comment:
Why are you removing this return value? For backwards compatibility, you have to deprecate it first.